app/graphql/inputs/user/login.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Input object for a userLogin mutation
+module Inputs
+  module User
+    class Login < Inputs::BaseInputObject
+      description "Credentials necessary for user login"
+      # the name is usually inferred by class name but can be overwritten
+      graphql_name "UserLoginInput"
+
+      argument :login, String, required: true
+      argument :password, String, required: true
+      argument :remember_me, Boolean, required: false
```

Comment:
I'm having trouble figuring out what "remember_me" does in this context!

In the old UI, it causes rails to set a cookie on the client browser.  If that cookie is present it automatically logs the user in if session_user is cleared.  That way, if you come back tomorrow after the session has expired, it will log you back in without you even noticing.

How would this look from a svelte perspective?  Would you have Svelte store the token in a cookie after logging in?  Then when you return to the site the next day and it doesn't have a token, it checks for a token in the "remember me" cookie and checks to see if that's valid, if so, you're automatically essentially logged in.  If that's right, then you would never actually send "remember_me" to the server at all.  It's all handled by svelte, and graphql doesn't need to know anything about it.

But if... something else... maybe graphql does need to know.  But as I see it right now, remember_me is not actually used anywhere.

Reply:
You're right, it doesn't do anything right now. And yes, Sveltekit stores the token in a local storage. 

I was thinking Rails would encode that param in the token as a flag to the frontend to either delete the token after x hours if the user forgets to log out (as in the case of a public or shared computer) -- or, if "remember me" is true, just keep using the same one. I believe the token is good for a while, unless we set the duration shorter. 

Am I still thinking garbled about tokens? Highly likely.... hopefully i'm starting to get it though...